src/sentry/api/utils.py

```diff
@@ -480,3 +480,19 @@ def duration(self):
             return self._duration
         else:
             return time.time() - self._start
+
+
+def id_or_slug_path_params_enabled(qual_name: str, organization_slug: str | None = None) -> bool:
+    # GA option
+    if options.get("api.id-or-slug-enabled"):
+        return True
+
+    # EA option for endpoints where organization is available
```

Comment:
I think we should clarify this, we're using the option not really as an EA of org endpoints but rather a check to gate to certain orgs if the slug is available. It would still need to pass the endpoint option check. Maybe something like:
```suggestion
    # EA option limiting to certain specific organizations for endpoints where organization is available
```